conference_scheduler/lp_problem/objective_functions.py

```diff
@@ -1,10 +1,30 @@
+from conference_scheduler.scheduler import schedule_to_array
+
+
 def capacity_demand_difference(slots, events, X, **kwargs):
     """
-    A function that minimises the difference between demand for an event and
+    A function that calculates the difference between demand for an event and
     the slot capacity it is scheduled in.
     """
     overflow = 0
     for row, event in enumerate(events):
         for col, slot in enumerate(slots):
             overflow += (slot.capacity - event.demand) * X[row, col]
     return overflow
+
+
+def number_of_changes(slots, events, original_schedule, X, **kwargs):
+    """
+    A function that minimises the number of changes between a given schedule
```

Comment:
Similar to your change on line 6: is this actually minimising the number of changes, or is this an objective function which I might use to drive a minimization? Looks to me like it’s just counting changes.